test-suite/prices.cpp

```diff
@@ -0,0 +1,218 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
+ Copyright (C) 2004 StatPro Italia srl
```

Comment:
This should attribute the copyright to either you or your employer — which one should it be?

Reply:
Fixed in e3939ca436998d568e664d269f4cf4f0d0783cc6.